src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs

```diff
@@ -874,10 +1058,12 @@ caughtException is OperationCanceledException ||
             }
         }
 
-        private sealed class MultiConnectSocketAsyncEventArgs : SocketAsyncEventArgs, IValueTaskSource
+        internal sealed class MultiConnectSocketAsyncEventArgs : SocketAsyncEventArgs, IValueTaskSource
```

Comment:
Why the change to `internal`? I don't see `MultiConnectSocketAsyncEventArgs` being used outside of `SocketAsyncEventArgs`.

Reply:
good catch. I was probably some early prototype that got refactored later.